unstructured/partition/html.py

```diff
@@ -57,9 +57,16 @@ def partition_html(
             raise ValueError(f"Expected content type text/html. Got {content_type}.")
 
         document = HTMLDocument.from_string(response.text)
-        elements = document.elements
 
     else:
         raise ValueError("Only one of filename, file, or text can be specified.")
 
+    elements: List[Element] = []
+    num_pages = len(document.pages)
+    for i, page in enumerate(document.pages):
+        for element in page.elements:
+            elements.append(element)
+        if include_page_breaks and i < num_pages - 1:
+            elements.append(PageBreak())
+
```

Comment:
This looks pretty similar to [the block in pdf.py](https://github.com/Unstructured-IO/unstructured/blob/feat/page-breaks/unstructured/partition/pdf.py#L138-L144), maybe worth factoring out to a function.

Reply:
Good call! I'll add a function to `common.py`

Reply:
Refactored and added to `common.py`